packages/atlas/src/components/FiltersBar/FiltersBar.tsx

```diff
@@ -96,7 +96,7 @@ export const FiltersBar: FC<ReturnType<typeof useFiltersBar> & FiltersBarProps>
   const datePopoverRef = useRef<PopoverImperativeHandle>(null)
   const lengthPopoverRef = useRef<PopoverImperativeHandle>(null)
   const othersPopoverRef = useRef<PopoverImperativeHandle>(null)
-  const { categories } = useCategories()
+  const { categories } = useQnCategories()
```

Comment:
Pending response from Adam on Slack whether the filtering shouldn't use display categories instead